### PR TITLE
202109 Add labels to indicate when running

### PR DESCRIPTION
**Pull request by owenThurm**
Tested with executable, not sure how this will look on Windows.

Should do a `git fetch` and `git checkout 202109_Bot_Round_Labels` and then run a round to check it out.

### CODE REVIEW

File: src/GUI/bot_gui.py [hunk 1]
```diff
@@ -226,6 +228,9 @@ def comment_round(self) -> None:
         ) = validated_data
 
         if username and password and num_comments and targets_list:
+            self.display_logging_in()
+            self.root.update_idletasks()
```
Comment (DanielGelfand) [inline src/GUI/bot_gui.py:232]: Nice, needed for GUI update

File: src/constants.py [hunk 2]
```diff
@@ -89,6 +89,10 @@
     "<a href=https://grwthapp.com/terms>terms of service</a>"
     "</p>"
 )
+
+LOGGING_IN_LABEL = "♻️   Logging in...  ♻️\n⚠️Do not shut off⚠️"
+COMMENTING_LABEL = "💬 Commenting...\n⚠️Do not shut off⚠️"
```
[the remark this answers is not in the document]
Comment (DanielGelfand) [reply src/constants.py:94]: Traceback (most recent call last):
  File ".\app.py", line 5, in <module>
    BotGUI().run()
  File "C:\Users\thedg\Documents\GUIBot\src\GUI\bot_gui.py", line 141, in run
    self.comment_round()
  File "C:\Users\thedg\Documents\GUIBot\src\GUI\bot_gui.py", line 247, in comment_round
    self.display_commenting()
  File "C:\Users\thedg\Documents\GUIBot\src\GUI\bot_gui.py", line 365, in display_commenting
    self.commenting_label = ttk.Label(self.root, text=COMMENTING_LABEL)
  File "C:\Users\thedg\AppData\Local\Programs\Python\Python36\lib\tkinter\ttk.py", line 761, in __init__
    Widget.__init__(self, master, "ttk::label", kw)
  File "C:\Users\thedg\AppData\Local\Programs\Python\Python36\lib\tkinter\ttk.py", line 559, in __init__
    tkinter.Widget.__init__(self, master, widgetname, kw=kw)
  File "C:\Users\thedg\AppData\Local\Programs\Python\Python36\lib\tkinter\__init__.py", line 2299, in __init__
    (widgetName, self._w) + extra + self._options(cnf))
_tkinter.TclError: character U+1f4ac is above the range (U+0000-U+FFFF) allowed by Tcl

[the remark this answers is not in the document]
Comment (DanielGelfand) [reply src/constants.py:94]: The commenting label will need custom for Windows.

[the remark this answers is not in the document]
Comment (owenThurm) [reply src/constants.py:94]: Just made a ticket for it

File: src/GUI/bot_gui.py [hunk 3]
```diff
@@ -240,6 +240,9 @@ def comment_round(self) -> None:
             )
 
             if ig_bot.failed_login:
+                self.logging_in_label.destroy()
```
Comment (DanielGelfand) [inline src/GUI/bot_gui.py:243]: Nice

**Review by DanielGelfand — APPROVED** [review]

**Pull request merged**